doc/connectivity/bluetooth/api/mesh/sar_cfg_cli.rst

```diff
@@ -0,0 +1,34 @@
+.. _bluetooth_mesh_sar_cfg_cli:
+
+SAR Configuration Client
+########################
+
+The SAR Configuration Client model is a foundation model defined by the Bluetooth mesh specification.
+It is an optional model, enabled with the :kconfig:option:`CONFIG_BT_MESH_SAR_CFG_CLI` configuration option.
+
+The SAR Configuration Client model is introduced in the `Bluetooth Mesh Protocol Specification version 1.1 <https://www.bluetooth.com/specifications/mesh-specifications/>`_,
```

Comment:
Same comment as above. I'm not sure the link is needed in every file.

Reply:
I'm thinking having the link once, the first time it is introduced on a page, cannot hurt.
At least if looking from the perspective of our new doc portal, where the user will search for specific topics rather than read through page by page in a traditional manner. Having the link available on the page that user reads, instead of having to go back to a page where the link was introduced is better. We are incouraged to have an "every page is a first page" philosophy.
But I can remove it if you feel strongly about it.
Remove or stay?

Reply:
What I **really** don't like is different styles in the same code base or documentation. What I expect will happen next is that here we will keep the link, but other pages will stay like they are right now (without the link). If we want to add link here, then we have to be consistent and add it to other pages as well (I don't see anything special in SAR models). 

I raised the concern of not having a common pattern/style for upstream docs on the last retrospective and created a follow up task NCSDK-23389. This particular case could be one of the points in that task.

P.S. This link doesn't really exist anymore, it redirects to all Bluetooth specs page.